parser/element_fill.go

```diff
@@ -312,6 +323,18 @@ func (f filler) setMap(field reflect.Value, node *Node) error {
 		return nil
 	}
 
+	if field.Type().Elem().Kind() == reflect.String {
+		for _, child := range node.Children {
+			if len(child.Children) > 0 {
+				f.fillRecursively(child.Name, child, field)
+			} else {
+				field.SetMapIndex(reflect.ValueOf(child.Name), reflect.ValueOf(child.Value))
+			}
```

Comment:
```suggestion
			f.fillRecursively(child.Name, child, field)
```

The case zero is already handled in fillRecursively method

Reply:
indeed 👍